pylint/checkers/stdlib.py

```diff
@@ -565,6 +573,7 @@ def visit_boolop(self, node: nodes.BoolOp) -> None:
     def visit_functiondef(self, node: nodes.FunctionDef) -> None:
         if node.decorators and isinstance(node.parent, nodes.ClassDef):
             self._check_lru_cache_decorators(node.decorators)
+            self._check_cache_decorators(node.decorators)
```

Comment:
The check message just above need to be completed too or we'll have trouble for  "functools-cache-decorating-method" when "lru-cache-decorating-method" is disabled.